_includes/releases/babelfish-1.2.1.md

```diff
@@ -0,0 +1,59 @@
+- Babelfish Version: 1.2.1
+- PostgreSQL Server Version: 13.7
+- Download source distributions:
+  - [BABEL_1_2_1__PG_13_7.zip](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_2_1__PG_13_7/BABEL_1_2_1__PG_13_7.zip)
+  - [BABEL_1_2_2__PG_13_6.tar.gz](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_2_1__PG_13_7/BABEL_1_2_1__PG_13_7.tar.gz)
```

Comment:
This should be updated: BABEL_1_2_2__PG_13_6.tar.gz

As should the date, on row 8